pkg/pillar/types/cipherinfotypes.go

```diff
@@ -48,3 +48,11 @@ type CipherBlockStatus struct {
 func (status *CipherBlockStatus) Key() string {
 	return status.CipherBlockID
 }
+
+type EncryptionBlock struct {
```

Comment:
This should have a comment (like elsewhere) that the fields used depends on the usage of the EncryptionBlock.

Reply:
Done..